### ISSUE TITLE
Control Data Shuffling and Set torch.backends Settings

### ISSUE THREAD

**Issue by ridmajayasundara** [cepdnaclk/e18-4yp-GPU-Acceleration-for-Deep-Learning-based-Comprehensive-ECG-analysis#50]
data_loader = DataLoader(dataset, **shuffle=True, worker_init_fn=np.random.seed(seed))**

torch.backends.cudnn.deterministic = True
torch.backends.cudnn.benchmark = False

https://chatgpt.com/share/efa8ba9a-9904-4f05-a7f4-d1068b43e248
